### PR TITLE
bump knmstate to 0.15.4

### PR DESCRIPTION
**Pull request by phoracek**
Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

**Comment by qinqon** [comment]
@phoracek is the lifecycle erro expected ?

**Comment by phoracek** [comment]
It is not. Investigating.

File: test/check/check.go [hunk 1]
```diff
@@ -404,8 +404,7 @@ func checkConfigCondition(conf *opv1alpha1.NetworkAddonsConfig, conditionType Co
 
 func describeAll() string {
 	description, err := Kubectl("-n", components.Namespace, "describe", "all")
-	Expect(err).ToNot(HaveOccurred())
-	return description
+	return fmt.Sprintf("description:\n%v\nerror:\n%v", description, err)
```
Comment (qinqon) [inline test/check/check.go]: unrelated ?
